### PR TITLE
Skip IK when IK fails in picking

### LINKED ISSUE #2366: Prepare for IK failure
Pickingの最中にIKが失敗しても、その点をスキップする or Pickingを中止することで、タスクを続けられるようにする

### PR DESCRIPTION
**Pull request by pazeshun**
Close #2366

### CODE REVIEW

File: jsk_arc2017_baxter/euslisp/lib/arc-interface.l [hunk 1]
```diff
@@ -763,6 +768,9 @@
                                  :rpy (float-vector pinch-yaw 0 0))
                     :move-palm-end t
                     :rotation-axis :z)))
+      (unless av
+        (ros::ros-info "[:try-to-pinch-object] arm:~a IK to object fails. Abort picking" arm)
```
Comment (wkentaro) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: Please use ros-error.

File: jsk_arc2017_baxter/euslisp/lib/arc-interface.l [hunk 2]
```diff
@@ -744,7 +749,7 @@
                     :move-palm-end t
                     :rotation-axis :z)))
       (unless av
-        (ros::ros-info "[:try-to-pinch-object] arm:~a IK fails. Abort picking" arm)
+        (ros::ros-info "[:try-to-pinch-object] arm:~a IK to object fails. Abort picking" arm)
```
Comment (wkentaro) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: Please use ros-error.

Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: Fixed

File: jsk_arc2017_baxter/euslisp/lib/arc-interface.l (hunk 1 above)
Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: Fixed

**Comment by pazeshun** [comment]
I couldn't test this because IK doesn't fail in my experiments...
Please merge this and test this continuously.

**Pull request merged**